### PR TITLE
Some improvements for network policy advisor

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
- Align "--namespace" and "--namespaces" flags
- Add --output option for report comman

### CODE REVIEW

**Comment by mauriciovasquezbernal** [comment]
@alban I think these changes are still relevant.

File: cmd/kubectl-gadget/networkpolicyadvisor.go [hunk 1]
```diff
@@ -158,6 +167,10 @@ func runNetworkPolicyMonitor(cmd *cobra.Command, args []string) {
 }
 
 func runNetworkPolicyReport(cmd *cobra.Command, args []string) error {
+	contextLogger := log.WithFields(log.Fields{
+		"command": "kubectl-gadget network-policy monitor",
```
Comment (alban) [inline cmd/kubectl-gadget/networkpolicyadvisor.go:171]:
```suggestion
func runNetworkPolicyReport(cmd *cobra.Command, args []string) error {
	contextLogger := log.WithFields(log.Fields{
		"command": "kubectl-gadget network-policy report",
```

File: cmd/kubectl-gadget/networkpolicyadvisor.go [hunk 2]
```diff
@@ -169,7 +182,21 @@ func runNetworkPolicyReport(cmd *cobra.Command, args []string) error {
 	}
 
 	advisor.GeneratePolicies()
-	fmt.Printf(advisor.FormatPolicies())
 
-	return err
+	w, closure, err := newWriter(outputFileName)
+	if err != nil {
+		contextLogger.Fatalf("Error creating file %q: %q", outputFileName, err)
```
Comment (alban) [inline cmd/kubectl-gadget/networkpolicyadvisor.go:188]: Does it look better with the following?

```suggestion
		contextLogger.Fatalf("Error creating file %q: %s", outputFileName, err)
```

Ditto in other contextLogger.Fatalf calls.

**Review by alban — APPROVED** [review]
LGTM

**Pull request merged**